SharpPcap/LibPcap/LibPcapSafeNativeMethods.Interop.cs

```diff
@@ -66,13 +66,15 @@ internal static partial class LibPcapSafeNativeMethods
         [DllImport(PCAP_DLL, CallingConvention = CallingConvention.Cdecl)]
         internal extern static void pcap_freealldevs(IntPtr /* pcap_if_t * */ alldevs);
 
+        /// <summary>
+        /// Open a generic source in order to capture / send (WinPcap only) traffic.
+        /// </summary>
         [DllImport(PCAP_DLL, CallingConvention = CallingConvention.Cdecl)]
-        internal extern static PcapHandle /* pcap_t* */ pcap_open(
+        internal extern static PcapHandle /* pcap_t* */ pcap_open_live(
             [MarshalAs(UnmanagedType.CustomMarshaler, MarshalTypeRef = typeof(PcapStringMarshaler))] string dev,
-            int packetLen,
-            int flags,
-            int read_timeout,
-            ref pcap_rmtauth rmtauth,
```

Comment:
pcap_open is the only function that supports remote authentication with RPCAP, using it is required.

It is supported in Linux since Libpcap 1.9

Reply:
I manually built libpcap 1.10.3 but didn't find `pcap_open` in the exported symbols.